docs/docs/creating-global-styles.md

```diff
@@ -2,7 +2,130 @@
 title: Creating global styles
 ---
 
-In this guide, we'll walk through using traditional global styles in your site.
+In nearly every site, there will be some global styles, such as a reset or typography defaults. This guide will walk through how to add global styles to your site, whether you use standard `.css` files (or with preprocessors like Sass/LESS) or a CSS-in-JS solution.
+
+## Table of Contents
+
+- [How to add global styles in Gatsby with standard CSS files](#how-to-add-global-styles-in-gatsby-with-standard-css-files)
+- [How to add global styles in Gatsby using CSS-in-JS](#how-to-add-global-styles-in-gatsby-using-css-in-js)
+- [Add global styles with CSS files and no layout component](#add-global-styles-with-css-files-and-no-layout-component)
+
+## How to add global styles in Gatsby with standard CSS files
+
+The best way to add global styles is with a [shared layout component](https://next.gatsbyjs.org/tutorial/part-three/#your-first-layout-component). This layout component is used for things that are shared throughout the site, including styles, header components, and other common items.
+
+> _NOTE:_ This pattern is implemented by default in [the default starter](https://github.com/gatsbyjs/gatsby-starter-default).
+
+To create a shared layout with global styles, start by creating a new Gatsby site with the [hello world starter](https://github.com/gatsbyjs/gatsby-starter-hello-world).
+
+```shell
+# Create the site
+gatsby new global-styles https://github.com/gatsbyjs/gatsby-starter-hello-world#v2
+```
+
+Open your new site in your code editor and create a new directory at `/src/components`. Inside, create two new files:
+
+```diff
+  global-styles/
+  └───src/
+      └───components/
++     │   │─  layout.js
++     │   └─  layout.css
+      │
+      └───page/
+          └─  index.js
+```
+
+Inside `src/components/layout.css`, add some global styles:
+
+```css
+div {
+  background: red;
+  color: white;
+}
+```
+
+In `src/components/layout.js`, include the stylesheet and export a layout component:
+
+```js
+import React from "react"
+import "./layout.css"
+
+export default ({ children }) => <div>{children}</div>
+```
+
+Finally, update `src/pags/index.js` to use the new layout component:
+
+```jsx
+import React from "react"
+import Layout from "../components/layout"
+
+export default () => <Layout>Hello world!</Layout>
+```
+
+Run `gatsby develop` and you’ll see the global styles applied.
+
+![Global styles](./images/global-styles.png)
+
+## How to add global styles in Gatsby using CSS-in-JS
+
+> **NOTE:** For this example, we’ll be using [Emotion](https://emotion.sh), but the implementation is similar for other CSS-in-JS solutions as well.
```

Comment:
The first "note" is italicized, this is bold. We don't have consistency across docs rn, but maybe pick one within this file?